zml/ops.zig

```diff
@@ -470,7 +469,8 @@ pub fn sort(
         inits[i * 2 + 1] = Tensor{ ._shape = arg_shape, ._id = undefined, ._donation = .no_buffer };
     }
     const ctx = CompilationContext.current();
-    const block = ctx.makeBlock(BodyS, &comp_fn, blkctx, inits);
+    const block, const cmp_res = ctx.makeBlock(BodyS, &comp_fn, blkctx, inits);
+    _ = cmp_res;
```

Comment:
```suggestion
    const block, _ = ctx.makeBlock(BodyS, &comp_fn, blkctx, inits);
```

(Except if you think it's nice to convey what is the second returned value)

Reply:
I don't think this work with our Zig version.

Reply:
my bad it works, updated every where applicable